relayer/relayer.go

```diff
@@ -136,7 +136,9 @@ func NewRelayer(
 	}
 
 	if shouldProcessMissedBlocks {
-		err = r.processMissedBlocks(sub)
+		// If StartBlockHeight is not set in the config, then we process from height 0
+		startHeight := big.NewInt(0).SetUint64(sourceSubnetInfo.StartBlockHeight)
+		err = r.processMissedBlocks(sub, startHeight)
```

Comment:
Would it make sense to to look up the configs/upgrades for this chain, and only start from the first block where warp was enabled?

Reply:
not sure how easy/hard it is to find that block, but seems to make sense to me.

Reply:
Depends on https://github.com/ava-labs/awm-relayer/pull/124, so created a separate issue to track here: https://github.com/ava-labs/awm-relayer/issues/129